tests/conftest.py

```diff
@@ -23,3 +24,9 @@ def captured_print():
     with io.StringIO() as output:
         with patch("builtins.print", new=lambda msg: output.write(f"{msg}\n")):
             yield output
+
+
+@pytest.fixture(scope="function")
+def clean_up_dvc():
+    yield
+    Repo(".").pull(force=True)
```

Comment:
Found the solution [here](https://github.com/iterative/dvc/issues/5999#issuecomment-840199287)